src/types/DbSubroutineInput.ts

```diff
@@ -11,6 +11,7 @@ export interface DbSubroutineUserDefinedOptions {
 
 export interface DbSubroutineSetupOptions {
 	userDefined?: DbSubroutineUserDefinedOptions;
+	requestId?: string;
```

Comment:
At first I was confused as to why this was added to this interface, but after discussing with @kthompson23 it made sense that eventually this `requestId` will make its way to the db server.

I think it might be beneficial to have a comment or something for now indicating that it is for future use by the db server logic as well as for the request header.